db-scheduler-ui-frontend/src/components/history/LogList.tsx

```diff
@@ -58,7 +86,10 @@ export const LogList: React.FC = () => {
       </HStack>
       <Accordion allowMultiple>
         {data?.map((log: Log) => (
-          <LogCard key={log.taskName + log.taskInstance + log.id} log={log} />
+          <LogCard
+            key={log.id + log.taskName + log.taskInstance + log.id}
```

Comment:
Hvorfor er log.id med to ganger? Trenger forsåvidt bare log.id og hvis den er unik kom jeg på nå